airflow-core/src/airflow/models/dagbag.py

```diff
@@ -536,7 +536,7 @@ def bag_dag(self, dag: DAG):
         except (AirflowClusterPolicyViolation, AirflowClusterPolicySkipDag):
             raise
         except Exception as e:
-            self.log.exception(e)
+            self.log.exception(str(e))
```

Comment:
Shouldn't be needed anymore
```suggestion
            self.log.exception(e)
```

(I mean, it's silly to log the exception string with the exception traceback, but lets not fix that here)